overlord/devicestate/devicemgr.go

```diff
@@ -511,7 +511,7 @@ func (m *DeviceManager) ensureInstalled() error {
 	}
 
 	// Note: thisalso stop auto-refreshes indirectly
```

Comment:
And here, missing a space.

Reply:
actually this comment was in the wrong place